ut/utils.cpp

```diff
@@ -203,6 +225,8 @@ std::ostream & operator<<(std::ostream & ostr, const ColumnValue& v) {
 
 }
 
+// map(low_cardinality(fixed_string(10)), array(nullable(uint64)))
+// {(key1, [1, 2, null, 4]), (key2, [null, 6, 7, 8])}
```

Comment:
can we remove this comment or add a more descriptive one to explain the function?